examples/c++/tca9548a.cxx

```diff
@@ -0,0 +1,90 @@
+ * 
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+ * LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include <unistd.h>
+#include <iostream>
+#include <signal.h>
+
+//#include "../mcp9808/mcp9808.hpp"
+#include "tca9548a.hpp"
+
+using namespace std;
+using namespace upm;
+
+/*
+ * This examble is built to multiplex channel 2 and Channel 4 bearing 
+ * sensors MCP9808
+ * call to the sensors in this example are commented
+ * because this example code must be self contained
+ * resources from another sensor cannot be used
+ * 
+ */
+
+int main(int argc, char **argv)
+{
+
+  // Instantiate a TCA9548A instance of i2c multiplexer
+  upm::TCA9548A *multiplex = new upm::TCA9548A(0);
+  
+  //**************************************
+  // Open i2c bus on multiplexer channel 4
+  //**************************************
+  multiplex->setPort(4,multiplex->ENABLED,multiplex->EXCLUSIVE);
+  // Instantiate a MCP9808 instance using multiplex C4 i2c bus and MCP9808 default address
+  //upm::MCP9808 *sensor1 = new upm::MCP9808(0);
+  // we show temperature
+  //cout << "Temperature: " << sensor1->getTemp()
+  //     << endl;
+  //cout << "Exiting..." << endl;
+
+  // Close com with channel 4 
+  multiplex->disableAllPorts();
+  
+  //**************************************
+  // Open i2c bus on multiplexer channel 2
+  //**************************************
+  multiplex->setPort(2,multiplex->ENABLED,multiplex->EXCLUSIVE);
+  // Instantiate a MCP9808 instance using multiplex C4 i2c bus and MCP9808 default address
+  //upm::MCP9808 *sensor2 = new upm::MCP9808(0);
+  //close com with channel 2
+  multiplex->disableAllPorts();
+  //*****************************************************
+  // Open 2 i2c bus on multiplexer channel 4 and channel2
+  //*****************************************************
+  multiplex->setPort(4,multiplex->ENABLED,multiplex->EXCLUSIVE);
+  multiplex->setPort(2,multiplex->ENABLED,multiplex->INCLUSIVE);
+  // we show temperature
```

Comment:
Could we get rid of these commented out parts?  Or, at least use a consistent commenting style, like '//'?